### PR TITLE
Add popup for cluster.

### PR DESCRIPTION
**Pull request by tienhoah**
- Remove dropdown for cluster site detail.
- Add and adjust custom leaflet styles.

### CODE REVIEW

File: src/leaflet_property.js [hunk 1]
```diff
@@ -55,3 +55,33 @@ export const clusterIconNumberStyles = `
   color: white;
```
Comment (brianbancroft) [inline src/leaflet_property.js]: Would it make sense to call this leaflet_properties?

Also, would it be better for organizations if it was under a `constants` folder?

File: src/components/LeafletMap.js [hunk 2]
```diff
@@ -332,6 +342,48 @@ class LeafletMap extends Component {
...
+
+    const popUpContent = `
+            <div style="${popUpContentStyles}" class="popup-wrapper">
+              ${popUpArray.join('')}
+            </div>
+        `;
+
+    const popup = L.popup()
+      .setLatLng(coordinates)
+      .setContent(`${popUpContent}`);
```
Comment (brianbancroft) [inline src/components/LeafletMap.js]: Do you need string interpolation here? Can't you just jam in the string as a variable?

File: src/components/LeafletMap.js [hunk 3]
```diff
@@ -332,6 +342,48 @@ class LeafletMap extends Component {
     return selectMarkerCluster;
   }
 
+  createPopup(markersData, coordinates) {
+    const popUpContentStyles =
+      markersData.length > 2
+        ? leafletProperty.popUpHigherStyles
```
Comment (brianbancroft) [inline src/components/LeafletMap.js]: Can you explain the choice of the term "popUpHigherStyles"?

File: src/components/LeafletMap.js [hunk 4]
```diff
@@ -127,13 +127,18 @@ class LeafletMap extends Component {
...
+      sidePanelOpen: prevSidePanelOpen,
+      siteDetail: prevSiteDetail
+    } = prevProps;
+    const { markersData, sidePanelOpen, siteDetail, popupOpen } = this.props;
     const {
       mapZoomLevel: prevMapZoomLevel,
       mapBoundingBoxCorner: prevMapBoundingBoxCorner
     } = prevState;
     const { mapZoomLevel, mapBoundingBoxCorner } = this.state;
+    const prevSiteDetailId = prevSiteDetail && prevSiteDetail.id;
```
Comment (brianbancroft) [inline src/components/LeafletMap.js]: _My kingdom for a safe navigation operator... One day_

🌮

**Comment by tienhoah** [comment]
closes #118

**Pull request merged**